src/main/kotlin/lotto/Application.kt

```diff
@@ -1,30 +1,30 @@
 package lotto
 
 import lotto.domain.Buyer
-import lotto.domain.Lottos
-import lotto.domain.Statistics
+import lotto.domain.LottoStatistics
+import lotto.domain.LottoTicket
+import lotto.domain.MatchingMachine
+import lotto.domain.result
 import lotto.view.InputView
 import lotto.view.ResultView
 
 object Application {
     @JvmStatic
     fun main(args: Array<String>) {
         val price = InputView.purchasePrice()
+        val purchasedCount = Buyer(price).purchasedCount
+        val lottoTickets = (1..purchasedCount).map { LottoTicket().getLottoNumbers() }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val lottoTickets = (1..purchasedCount).map { LottoTicket() }
```